.styleci.yml

```diff
@@ -1,7 +1,5 @@
 preset: laravel
 
-linting: true
```

Comment:
You might need to rebase this with master, I added this in a [previous PR.](#676 )